.github/workflows/build-test.yaml

```diff
@@ -29,10 +29,6 @@ jobs:
           username: ${{secrets.IRON_BANK_ROBOT_USERNAME}}
           password: ${{secrets.IRON_BANK_ROBOT_PASSWORD}}
 
-      - name: Build the custom k3s image
-        if: ${{matrix.image}} != "rancher/k3s"
-        run: uds run build-image --set VERSION=${{matrix.version}} --no-progress
```

Comment:
if we are keeping the uds task in the `tasks.yaml` in as a comment, should we also keep this around but commented out?

Reply:
Good call - will update. Initially I had started tearing out pieces but think its beneficial to leave.

Reply:
Re-added, commented out!